src/ucp/api/ucp_compat.h

```diff
@@ -0,0 +1,36 @@
+/**
```

Comment:
For each API change is it important to generate the actual pdf and do visual inspection of the document.